### DISCUSSION TITLE
There is no allowed apps

### DISCUSSION THREAD

**Discussion by Oval-Otval** [schwabe/ics-openvpn#1601] [Q&A]
there are no allowed applications in the application header and I have absolutely no idea why. Help me with that please)
Smartphone Realme X3, application version from 14th December.
I'm new to github, so please be understanding

![Screenshot_2023-03-29-16-14-36-07_d1c130322af4a604846ad4d63b2dad88](https://user-images.githubusercontent.com/129302246/228569821-ba2b819c-b063-408e-8cbd-a1c25b97fac5.jpg)

**Comment by schwabe** [comment]
What version of the App are you running and what API version is your phone one? The first line from the log window will show those information.

**Comment by Oval-Otval** [comment]
This setting ended up in a different menu...
It's embarrassing. The issue is resolved.
![IMG_20230329_180128](https://user-images.githubusercontent.com/129302246/228598668-58b35126-31f4-4a43-8342-7029b09b4a90.jpg)
